docs/api/api-specification.md

```diff
@@ -0,0 +1,2170 @@
+                "url": "ombCategory",
+                "valueCoding": {
+                  "system": "urn:oid:2.16.840.1.113883.6.238",
+                  "code": "2135-2",
+                  "display": "Hispanic or Latino"
+                }
+              },
+              {
+                "url": "text",
+                "valueString": "Hispanic or Latino"
+              }
+            ],
+            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity"
+          },
+          {
+            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex",
+            "valueCode": "F"
+          },
+          {
+            "url": "http://saraalert.org/StructureDefinition/preferred-contact-method",
+            "valueString": "E-mailed Web Link"
+          },
+          {
+            "url": "http://saraalert.org/StructureDefinition/symptom-onset-date",
+            "valueDate": "2020-05-16"
+          },
+          {
+            "url": "http://saraalert.org/StructureDefinition/last-exposure-date",
+            "valueDate": "2020-05-11"
+          },
+          {
+            "url": "http://saraalert.org/StructureDefinition/isolation",
+            "valueBoolean": true
+          },
+          {
+            "url": "http://saraalert.org/StructureDefinition/full-assigned-jurisdiction-path",
+            "valueString": "USA, State 1"
+          }
+        ],
+        "active": true,
+        "name": [
+          {
+            "family": "mctest",
+            "given": [
+              "Lakeesha28",
+              "testy"
+            ]
+          }
+        ],
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "(333) 333-3333",
+            "rank": 1
+          },
+          {
+            "system": "phone",
+            "value": "(333) 333-3333",
+            "rank": 2
+          },
+          {
+            "system": "email",
+            "value": "7858879250fake@example.com",
+            "rank": 1
+          }
+        ],
+        "birthDate": "1971-01-21",
+        "address": [
+          {
+            "line": [
+              "3718 Nestor Unions"
+            ],
+            "city": "Port Leigh",
+            "state": "Massachusetts",
+            "postalCode": "00423-5596"
+          }
+        ],
+        "communication": [
+          {
+            "language": {
+              "coding": [
+                {
+                  "system": "urn:ietf:bcp:47",
+                  "code": "en",
+                  "display": "English"
+                }
+              ]
+            }
+          }
+        ],
+        "resourceType": "Patient"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+#### GET `[base]/QuestionnaireResponse?subject=Patient/[:id]`
+You can use search to find Monitoree daily reports by using the `subject` parameter.
+
+<a name="search-questionnaire-subj"/>
+
+GET `[base]/QuestionnaireResponse?subject=Patient/[:id]`
```

Comment:
This section with the alternating `#### Get` and `Get` ends up looking a little funky: 
![image](https://user-images.githubusercontent.com/52747882/102382226-357e9880-3f98-11eb-9f62-e5aa131cf979.png)

Technically I don't think there is anything explicitly wrong with the usage of headers here, it just looks a bit off with how this theme styles things.

Reply:
Due to the header "promotions" I think this looks better now, but let me know. https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b